### ISSUE TITLE
(Oldie but a goodie) Just updated to ubuntu 14.04 and now can't switch over to linux any more

### ISSUE THREAD

**Issue by GuillermoArjun** [dnschneid/crouton#1331]
Good morning/afternoon/evening,

To be safe here's my whole output following the 'common issues' checklist:

crosh> shell
chronos@localhost / $ sudo edit-chroot -a
precise
chronos@localhost / $ sudo sh ~/Downloads/crouton -n precise -u
Downloading latest crouton installer...
######################################################################## 100.0%
/usr/local/chroots/precise already exists; updating it...
Preparing chroot environment...
Installing brightness into the chroot...
Installing croutonpowerd into the chroot...
Installing croutonversion into the chroot...
Installing host-dbus into the chroot...
Installing host-x11 into the chroot...
Installing pulseaudio-default.pa into the chroot...
Installing croutoncycle into the chroot...
Installing croutonxinitrc-wrapper into the chroot...
Installing setres into the chroot...
Installing xinit into the chroot...
Installing xbindkeysrc.scm into the chroot...
Installing xserverrc into the chroot...
Installing xserverrc-xorg into the chroot...
Installing xserverrc-local.example into the chroot...
Installing startunity into the host...
Installing crouton-noroot into the chroot...
Installing startunity into the chroot...
Installing gnome-session-wrapper into the chroot...
Installing crouton-unity-autostart into the chroot...
Installing unity-autostart.desktop into the chroot...
Installing unity-profiled into the chroot...
Installing enter-chroot into the host...
Installing delete-chroot into the host...
Installing edit-chroot into the host...
Installing mount-chroot into the host...
Installing unmount-chroot into the host...
Installing crash_reporter_wrapper into the host...
Setting up ubuntu-xboxdrv (20150105-1) ...
/usr/bin/update-desktop-database
Warning in file "/usr/share/applications/gnumeric.desktop": usage of MIME type "zz-application/zz-winassoc-xls" is discouraged ("zz-application/zz-winassoc-xls" should be replaced with "application/vnd.ms-excel")
start: Unknown job: xboxdrv
dpkg: error processing package ubuntu-xboxdrv (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 ubuntu-xboxdrv
Failed to complete chroot setup.
Not unmounting /mnt/stateful_partition/crouton/chroots/precise as another instance is using it.

As a brief background into what I'd just done on my linux side. First I had a system update which I ran and then in order to run steam did the following...

I'd just followed the instructions on - https://help.ubuntu.com/community/InstallingCompilers and had run the following commands: 

sudo apt-get update
sudo apt-get upgrade
sudo apt-get install build-essential
gcc -v
make -v
sudo apt-get install libc6-dev
sudo apt-get install g++-multilib
sudo apt-get install ia32-libs

Then did a fresh install of steam, and then to get controller support, followed the instructions here: http://www.omgubuntu.co.uk/2014/06/ubuntu-xbox-controller-support-xboxdrv-driver

and ran the following:
sudo apt-add-repository ppa:rael-gc/ubuntu-xboxdrv
sudo apt-get update && sudo apt-get install ubuntu-xboxdrv

All was hunky dory until I clicked out of a window and then linux partition became unresponsive so I restarted the computer, and now I can't access any more.

Any help would be much appreciated.

Will

**Comment by GuillermoArjun** [comment]
I DID IT!
*typed from the linux partition*
Unnecessarily proud right now, but very very very necessarily proud of the fantastic community here.

For those of you interested in how this tale was resolved, the final steps that seemed to do it were:
I entered chroot to remove/purge xboxdrv
Tried deleting chroot to no avail
Ran the update one more time just to be safe and then it ran perfectly

Needless to say xboxdrv is staying firmly uninstalled!
